themes/base.theme.bash

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+# shellcheck shell=bash
```

Comment:
Just chiming in to confirm that we need to remove the 1st line here, leaving only the shellcheck statement.
(and of course also reverting the changes to `dot-bash.sh`